encoding/legacy-mb-tchinese/big5/big5-decoder.js

```diff
@@ -46,7 +46,7 @@ function big5Decoder(stream) {
 			if ((byte >= 0x40 && byte <= 0x7e) || (byte >= 0xa1 && byte <= 0xfe))
 				ptr = (lead - 0x81) * 157 + (byte - offset);
 			// "If there is a row in the table below whose first column is pointer, return the two code points listed in its second column"
-			switch (ptr) {
+			switch (ptr + "") {
```

Comment:
I'd prefer to convert the `case` statements to use numbers rather than strings, unless there's some reason I'm missing.